src/routes/use-ticket.ts

```diff
@@ -0,0 +1,81 @@
+import asyncHandler from 'express-async-handler';
+import httpErrors from 'http-errors';
+import { Router } from 'express';
+import { sanitizeString } from '../lib/DataValidator';
+import { getEventData, verifyUserInEvent, checkinUser, verifyHost } from '../lib/UserEventHelper';
+import { sendNotification } from '../lib/NotificationHelper';
+
+const router = Router();
+
+/**
+ * Marks a ticket as used after scanning
+ *
+ * Sample JSON POST
+ {
+    "eid": "8UhyXDgfoBAJBboJkMQQ",
+    "uid": "hAYk9Cyrqf7253kfOOSe" -> User ID of the user who's ticket you are validating
+ }
+ */
+router.post(
+  '/',
+  asyncHandler(async (req, res, next) => {
+    const respTitle = 'Ticket Scanner';
+    let respMessage = '';
+
+    const user = sanitizeString(req.body.user);
+    // const user = await getUserFromRequest(req);
+    const eid = sanitizeString(req.body.eid);
+    const uid = sanitizeString(req.body.uid);
+
+    // Validate the user who sent the request is the host of the event
+    try {
+      await verifyHost(eid, uid);
```

Comment:
Theres a already function called `validateHost` in `Event.ts`, you can use that instead of `verifyHost`. Upto you

Reply:
nice didn't see that one